### PR TITLE
Make audit folder buildable

### PR DESCRIPTION
**Pull request by hugihlynsson**

### CODE REVIEW

File: pages/audit/branch-audit.mdx [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
 
 The branch audit reviews the changes you have made on your branch and flags issues which are introduced by your changes, such as unexpected naming or inconsistent types — helping you keep your tracking plan clean and consistent.
 
-When <Link passHref href="/workspace/branches#review-branch-changes"><a>reviewing changes on a branch</a></Link>, you can see issues directly on your tracking plan items and in the instance of inconsistent casing you can automatically fix the issue by the click of a button.
+When [reviewing changes on a branch](/workspace/branches#review-branch-changes), you can see issues directly on your tracking plan items and in the instance of inconsistent casing you can automatically fix the issue by the click of a button.
 
 <center>
   <img
-    src={require('../../images/audit/inline-audit.png')}
+    src={'../../images/audit/inline-audit.png'}
```
Comment (bragur) [inline pages/audit/branch-audit.mdx:13]: These will be so:
```suggestion
    src={'../images/audit/inline-audit.png'}
```

File: pages/audit/overview.mdx [hunk 2]
```diff
@@ -1,21 +1,17 @@
...
 
-Furthermore, all changes on branches are also checked in the <Link passHref href="/audit/branch-audit"><a>Branch Audit</a></Link> to see if you have introduced any new issues on your branch.
+Furthermore, all changes on branches are also checked in the [Branch Audit](/audit/branch-audit) to see if you have introduced any new issues on your branch.
 
-To learn more about the audit rules, see <Link passHref href="/audit/rules"><a>Avo's tracking plan audit rules</a></Link>.
+To learn more about the audit rules, see [Avo's tracking plan audit rules](/audit/rules).
 
-> **Note:** The tracking plan audit only audits the _plan_. If you want to identify issues in your current tracking implementation and data streams, check out <Link passHref href="/workspace/inspector"><a>Tracking Observability</a></Link>.
+> **Note:** The tracking plan audit only audits the _plan_. If you want to identify issues in your current tracking implementation and data streams, check out [Tracking Observability](/workspace/inspector).
```
Comment (bragur) [inline pages/audit/overview.mdx:13]: You could also use the `Callout` component here. It’s a bit more nice.

**Review by bragur — APPROVED** [review]
LGTM! I didn’t run the code, but changes look legit.

File: pages/audit/overview.mdx (hunk 2 above)
Comment (hugihlynsson) [reply pages/audit/overview.mdx:13]: Hey nice!

**Pull request merged**